src/main/java/org/folio/search/service/setter/bibframe/authority/BibframeAuthorityTypeProcessor.java

```diff
@@ -0,0 +1,18 @@
+package org.folio.search.service.setter.bibframe.authority;
+
+import org.folio.search.domain.dto.BibframeAuthority;
+import org.folio.search.service.setter.FieldProcessor;
+import org.springframework.stereotype.Component;
+
+@Component
+public class BibframeAuthorityTypeProcessor implements FieldProcessor<BibframeAuthority, String> {
+
+  @Override
+  public String getFieldValue(BibframeAuthority bibframe) {
+    var type = bibframe.getType();
+    if (type == null) {
+      return null;
+    }
+    return type.toString();
```

Comment:
Sonar is showing a violation in this line -> https://sonarcloud.io/project/issues?id=org.folio%3Amod-search&pullRequest=605&resolved=false&sinceLeakPeriod=true